mod/gc_communities/views/default/js/stream_community_feed.js

```diff
@@ -0,0 +1,80 @@
+/**
+* stream_community.js
+*
+* Streaming Community JS - Listens on the community page for new community items. It then loads the new community items to the DOM
+*
+* @author Nick github.com/piet0024
+* @author Ilia github.com/phanoix
+* @author Mark github.com/markwooff
+**/
+
+$(document).ready(function(){
+    community_feed_stream_count(); 
+});
+
+// This function counts 2 minutes while a streaming page is loaded
+var interval = null;
```

Comment:
You're adding this to the global/window scope. Might want to wrap this all in an IIFE